lib/translator.h

```diff
@@ -0,0 +1,26 @@
+#ifndef LIB_TRANSLATOR_H
+#define LIB_TRANSLATOR_H
+
+#include "scsi_defs.h"
+#include "../third_party/spdk_defs/nvme_defs.h"
+
+namespace translator {
+struct TranslatedScsiStatus {
```

Comment:
Thoughts on calling this just "ScsiStatus" rather than "TranslatedScsiStatus"? It seems that the general use case for this structure is more generic than only holding translated data & name is more succinct

Reply:
I'd be happy to go with `ScsiStatus`. My only concern is that that might be confused with `scsi_defs::Status` https://github.com/googleinterns/scsi2nvme/blob/master/lib/scsi_defs.h#L29

Reply:
Ultimately decided to name this ScsiStatus. The different namespaces should be a sufficient differentiator.